doc/user-guide/components.rst

```diff
@@ -245,7 +245,15 @@ must put a link in front of it, see the `example that replaces
 generators and storage units with fundamental links and stores
 <https://pypsa.readthedocs.io/en/latest/examples/replace-generator-storage-units-with-store.html>`_.
 
-
+The ``marginal_cost`` of a Store apply to both the charging and the discharging.
+In the case of a cyclic store without losses, these costs would balance out to
+zero. This is different to the ``StorageUnit`` where the marginal cost apply to the
```

Comment:
Minor remark: Maybe better to stick to the phrases charging and discharging, like at the beginning of the paragraph? Meaning that instead of writing marginal cost of production, write `marginal cost of discharge`

Reply:
Yes, you are right. I have added it in brackets, so that it fits to the documentation of the storage unit which refers to cost of production and the previous paragraph which refers to charging/discharging